docs/UserGuide.md

```diff
@@ -5,44 +5,111 @@ title: User Guide
+--------------- | ---------------
+:heavy_exclamation_mark: | Indicates warnings (take caution when you see this symbol)
+:information_source: | Important information to take note about
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` installed on your computer.
+
+To check if Java `11` is currently installed, you may follow this short [guide](https://www.baeldung.com/java-check-is-installed).
+Otherwise, if Java `11` is not installed, you may follow the installation instructions over 
+[here](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html#GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A).
+
+2. Next, download the latest `gitgud.jar` from [here](https://github.com/AY2122S1-CS2103T-W13-4/tp/releases).
 
-2. Download the latest `gitgud.jar` from [here](https://github.com/AY2122S1-CS2103T-W13-4/tp/releases).
 
 3. Copy the file to the folder you want to use as the _home folder_ for your gitGud application.
 
+
 4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds.<br>
-   ![Ui](images/ui-snapshots/startup.png)
-<br><ins>gitGud upon startup.</ins>
-5. Type the command in the command box and press `Enter` to execute it. e.g. typing **`help`** and pressing `Enter` will open the help window.<br>
-   Some example commands you can try:
+   ![Ui](images/ui-screenshots/startup.png)
+<br><center><ins>Image: gitGud upon startup.</ins></center>
+
+5. Choose and type in a command into the command input box and press `Enter` to execute it. 
+e.g. typing **`help`** and pressing `Enter` will open the help window.<br>
+   Here are some example commands you can try:
 
    * **`friend --list`** : Lists all friends.
 
-   * **`friend --add`**`Draco -n Marcus` : Adds a friend __Marcus__ with gitGud `FRIEND_ID` of __Draco__ to the application.
+   * **`friend --add Draco -n Marcus`** : Adds a friend __Marcus__ with gitGud `FRIEND_ID` of __Draco__ to the application.
 
-   * **`friend --delete`**`Draco` : Deletes friend with gitGud `FRIEND_ID` of __Draco__.
+   * **`friend --delete Draco`** : Deletes friend with gitGud `FRIEND_ID` of __Draco__.
 
    * **`exit`** : Exits the app.
-
-6. Refer to the [Features](#features) below for details of each command.
+   
+Congratulations! You have successfully set up gitGud.
+For more in-depth details of each command, refer to the [features](#features) section below.
 
 --------------------------------------------------------------------------------------------------------------------
+## gitGud User Interface 
+
+Before we jump into the commands, let us take a moment to familiarise ourselves with the user interface. 
 
+gitGud provides a gamer-themed user interface as shown in the image below with 6 main sections.
+
+![Ui-labelled](images/ui-screenshots/ui-overview.png)
```

Comment:
I think white on bright green doesn't have enough contrast to be easily readable. If possible, consider changing it, but its very low priority.

Reply:
I agree with this, it can potentially be hard a little to see. But I think the image looks really great other than that! great work.